client/src/utils/uniformize-challenge-size.ts

```diff
@@ -0,0 +1,33 @@
+import { ChallengeFiles } from '../redux/prop-types';
+
+interface UniformizeChallengeSizeProps {
```

Comment:
```suggestion
interface UniformizeChallengeSizeArgs {
```
Nit: nothing React-related about this.